src/constants.R

```diff
@@ -2,18 +2,52 @@
 main_api_url <- "http://avadar.novel-t.ch/argus-r/sesdashboard/web/"
 user_name <- Sys.getenv("api_user_name")
 api_password <- Sys.getenv("api_password")
-langueage <-  Sys.getenv("api_locale")
+language <-  Sys.getenv("locale")
```

Comment:
are these envvar names like `api_user_name` pre-existing or have we added them recently? If it's us who added them, I suggest renaming all these variables to sth like `ARGUS_DASHBOARD_API_USER_NAME`, `ARGUS_DASHBOARD_LOCALE`

It's a convention to name env vars in all caps
Also, since they are often global on the system, it's good to add very specific name to avoid name conflicts

Reply:
done